magento-research/pwa-studio
# Wrapped Peregrine Context Provider Component

## Description
Add a Peregrine component that wraps the composition of context providers needed to use Peregrine hooks (currently just WindowSize and Toasts).

## Related Issue





Closes #1312

## Verification Steps

1. Go to Home page and use React DevTools to inspect source
2. Verify that `<App />` is wrapped by `WindowSizeContextProvider` and `ToastContextProvider`

## How have YOU tested this?



Ran verification steps and ran jest

## Screenshots / Screen Captures (if appropriate):

## Proposed Labels for Change Type/Package




## Checklist:
